### PR TITLE
Adding Mish Activation

### PR DESCRIPTION
**Pull request by thebhatman**
This PR implements the Mish Activation and resolves https://github.com/PaddlePaddle/Paddle/issues/21219

### CODE REVIEW

**Comment by thebhatman** [comment]
I am not sure why the CI builds are still yellow and loading... Also do I have to do something in python folder to allow for python imports of Mish?

**Comment by luotao1** [comment]
https://www.paddlepaddle.org.cn/documentation/docs/en/advanced_usage/development/contribute_to_paddle/local_dev_guide_en.html#commit

Sorry for the late reply, please add `test=develop` in commit information to trigger the CI.

File: paddle/fluid/inference/anakin/convert/activation.cc [hunk 1]
```diff
@@ -61,4 +61,5 @@ void ActivationOpConverter<TargetT, PrecisionT>::operator()(
 REGISTER_ANAKIN_OP_CONVERTER(sigmoid, SigmoidOpConverter);
 REGISTER_ANAKIN_OP_CONVERTER(tanh, TanhOpConverter);
 REGISTER_ANAKIN_OP_CONVERTER(swish, SwishOpConverter);
+REGISTER_ANAKIN_OP_CONVERTER(mish, MishOpConverter);
```
Comment (luotao1) [inline paddle/fluid/inference/anakin/convert/activation.cc]: You don't need to add mish in Anakin, since it doesn't support it.

Comment (thebhatman) [reply paddle/fluid/inference/anakin/convert/activation.cc]: So shall I remove Mish from Anakin completely in all files that come under anakin?

Comment (thebhatman) [reply paddle/fluid/inference/anakin/convert/activation.cc]: Done.

**Comment by luotao1** [comment]
> Also do I have to do something in python folder to allow for python imports of Mish

Yes, you should add python wrapper in `python/paddle/fluid/layers/nn.py`, like some activations_op such as `relu6`, `elu` etc.

Besides, you should add unit-tests in `python/paddle/fluid/tests/unittests/test_activation_op.py`

**Comment by luotao1** [comment]
What's the python wrapper of mish？If it is `paddle.fluid.layers.mish(x, name=None)`, you could follow the way `paddle.fluid.layers.abs(x, name=None)` does, and don't need to add python wrapper in `python/paddle/fluid/layers/nn.py`

**Comment by thebhatman** [comment]
I have added the python wrapper for Mish now.

**Comment by thebhatman** [comment]
Can I add Mish in tensorrt/convert and operators, and the python wrappers together in one PR? That would basically be adding the whole Mish module in one PR.

**Comment by luotao1** [comment]
Yes, you can, but the review process will be longer.

**Pull request closed without merging**